src/fullyautomatednutcracker/cogs/filters.py

```diff
@@ -901,7 +928,80 @@ async def commie(self, ctx):
     @commands.command(aliases=["gay", "rainbow", "mod"])
     async def gaygaygay(self, ctx):
         await self.apply_mask(ctx, "gaygaygay.png")
-
+        
+        
+    @commands.command(
+        name = "trumptweet",
+        aliases = ['trump', 'trumpet']        
+    )
+    async def djtj(self, ctx, *, text = None):
+        """Trump tweets whatever you say."""
+        
```

Comment:
Add the following here provided you want to support what was listed in the later remark:

```
arguments = await self.get_args_from_message(ctx)
if not arguments:
    return
```

Reply:
wait, do i also replace text with arguments, or leave the rest of the function as-is?